js/copy-service/Evaluator/Evaluator.test.ts

```diff
@@ -35,23 +36,24 @@ describe('Evaluator', () => {
   describe('getCached', () => {
     test('gets the result from the evaluation cache', () => {
       const result = { some: 'result' };
-      const ast = { an: 'ast' };
+      const ast = new SyntaxNode();
 
       evaluator.evaluationCache.set(ast, result);
-      expect(evaluator.getCached(ast)).toBe(result);
+      expect(evaluator.getCached((ast))).toBe(result);
```

Comment:
Extra parens left over
```suggestion
      expect(evaluator.getCached(ast)).toBe(result);
```